claat/parser/md/parse.go

```diff
@@ -406,11 +406,23 @@ func handleInfobox(ps *parserState) {
 
 // handleImage handles <img> tags. It assumes the tokenizer is pointing to the <img> tag itself.
 func handleImage(ps *parserState) {
+	var n *types.ImageNode
+	var alt, title string
 	for _, v := range ps.t.Attr {
 		if v.Key == "src" {
-			ps.emit(types.NewImageNode(v.Val))
-			break
+			n = types.NewImageNode(v.Val)
+		}
+		if v.Key == "alt" {
```

Comment:
I'd rather use switch, which won't go into all the checks like these `if`s do. Also, note `strings.ToLower`:

```go
switch strings.ToLower(v.Key) {
case "src":
  n = types.NewImageNode(v.Val)
case "alt":
  alt = v.Val
case "title":
  title = v.Val
}
```